data/us/ohio/ohio.csv

```diff
@@ -5,7 +5,7 @@ id,name,brewery_type,street,city,state,postal_code,website_url,phone,created_at,
 4kd-crick-brewery,4kd Crick Brewery,brewpub,211 Carpenter Rd,Defiance,Ohio,43512-1718,http://Www.4kdcrickbrewery.com,4199562863,33:59.4,43:21.0,United States,-84.341647,41.296252,
 actual-brewing-company-llc,"Actual Brewing Company, LLC",micro,655 N James Rd,Columbus,Ohio,43219-1837,http://wwww.actualbeer.com,6146363825,33:59.4,43:21.5,United States,-82.9080986,39.9868523,
 akronym-brewing-llc,Akronym Brewing LLC,micro,58 E Market St,Akron,Ohio,4430,http://www.akronymbrewing.com,3306208274,33:59.4,43:22.1,United States,-81.51645882,41.08512193,
-alematic-artisan-ales,Alematic Artisan Ales,planning,,Huber Heights,Ohio,45424-3857,http://www.alematicbrewing.com,9376122337,33:59.4,38:55.6,United States,,,
+alematic-artisan-ales,Alematic Artisan Ales,planning,,Huber Heights,Ohio,45424-3857,http://www.alematicbrewing.com,9376122337,33:59.4,38:55.6,United States,,39.12642354,

```

Comment:
You are a legend! Any chance we could get a longitude on this one? Thank you!

Reply:
Oh my mistake! I went ahead and updated it and also updated the type as well since it's no longer in the planning stage! Hmm to be honest I'm not sure how to edit my previous commit like I resolved the conflict for the other pr but I made the changes in the csv! I'll add that here and let me know if there's something else I should do instead!

Reply:
One thing you can try is to upload the updated CSV with the same filename to overwrite and choose the same PR branch. The PR should update with the new data.

If that doesn't work out, then we can close this PR and open another one or merge this one and then make an edit. Let me know which you're most comfortable with. 

Thanks!